.github/workflows/ci-nix.yml

```diff
@@ -159,7 +159,7 @@ jobs:
   ccov:
     name: "Code coverage"
     runs-on: buildjet-8vcpu-ubuntu-2004
-    timeout-minutes: 30
+    timeout-minutes: 45
```

Comment:
Does this happen when something isn't cached? How did it become part of this PR?